packages/data-portal-explore/src/components/FileFilterControls.tsx

```diff
@@ -151,6 +152,31 @@ export const FileFilterControls: React.FunctionComponent<IFileFilterControlProps
                             });
                     }}
                 />
+                <FilterDropdown
+                    {...dropdownProps}
+                    attributes={[AttributeNames.viewers]}
+                    options={(attrName: AttributeNames) => {
+                        return options(attrName).map((e: OptionType) => {
+                            const viewerLabels = {
+                                [FileViewerName.minerva]: 'Minerva',
+                                [FileViewerName.ucscXena]: 'USCS Xena',
+                                [FileViewerName.cellxgene]: 'CellxGene',
+                                [FileViewerName.isbcgc]: 'BigQuery',
+
+                                // excluded or aggregated values:
+                                // we are not supposed to see these as filter options
+                                [FileViewerName.autoMinerva]: 'Auto Minerva', // aggregated as Minerva
+                                [FileViewerName.customMinerva]:
+                                    'Custom Minerva', // aggregated as Minerva
+                                [FileViewerName.cds]: 'CDS', // excluded (this only appears as a download source)
+                                [FileViewerName.idc]: 'IDC', // excluded (we do not show IDC links anymore)
```

Comment:
we can probably completely drop support for IDC and remove all related files & code